### PR TITLE
[PF-205] add cursor pointer to accordion component, replace all `${}`…

### PR DESCRIPTION
**Pull request by awicone**
… with classNames

### CODE REVIEW

File: src/components/Accordion/Accordion.module.scss [hunk 1]
```diff
@@ -1,55 +1,18 @@
 @use '../../assets/styles/theme';
 
-.card-header {
```
Comment (tastytea-github) [inline src/components/Accordion/Accordion.module.scss:3]: Why you have removed these bunch of CSS classes? Won't these changes break the design?

Comment (awicone) [reply src/components/Accordion/Accordion.module.scss:3]: No. It was unnecessary

File: src/components/Timeline/TimelineCards/TimelineCard.tsx [hunk 2]
```diff
@@ -29,48 +29,44 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
         }
     }
 
+    const timelineStyle = classNames(
+        s.cdTimelineBlock,
+        { [s.reverse]: id % 2 === 0 }
+    )
+
+    const cardContainer = classNames(
```
Comment (tastytea-github) [inline src/components/Timeline/TimelineCards/TimelineCard.tsx:37]: You can add style or className postfixe for the next constants like you did for const timelineStyle to understand it is a style/className

File: src/components/Timeline/TimelineCards/TimelineCard.tsx [hunk 3]
```diff
@@ -29,48 +29,44 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
...
+        s.cardContainer,
+        { [s.showBack]: showBack }
+    )
+
+    const cdTimelineContent = classNames(
+        s.cdTimelineContent,
+        id % 2 === 0 ? s.arrowLeft : s.arrowRight,
+        s[cardInfo.size]
```
Comment (tastytea-github) [inline src/components/Timeline/TimelineCards/TimelineCard.tsx:45]: Looks tricky. If you pass size that has not the same class in CSS then it wouldn't be applied and you won't catch any error about it.

File: src/components/TreeInfoBlock/TreeInfoBlock.tsx [hunk 4]
```diff
@@ -7,47 +7,45 @@ import cacaoTreeImage from '@/assets/images/view-tree/cacao-viewer.png'
...
+              </div>
+            )}
           </div>
         </div>
-        <div className={s.treeInRound}>
-          <img src={treeIcons[treeNames.indexOf(treeData.treeType.toUpperCase())]} className={s.treeImage} alt='tree' />
-        </div>
+      </div>
+      <div className={s.treeInRound}>
+        <img src={treeIcons[treeNames.indexOf(treeData.treeType.toUpperCase())]} className={s.treeImage} alt='tree' />
```
Comment (tastytea-github) [inline src/components/TreeInfoBlock/TreeInfoBlock.tsx:48]: too long value treeIcons[treeNames.indexOf(treeData.treeType.toUpperCase())] Would be nice to move it to const.

File: src/components/Timeline/TimelineCards/TimelineCard.tsx [hunk 5]
```diff
@@ -23,69 +23,89 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
 
     const [showBack, setIsShowBack] = useState<boolean>(false)
 
+    switch (cardInfo.size) {
+        case 'md': s.md
+            break
+        case 'xs': s.xs
+            break
+    }
```
Comment (tastytea-github) [inline src/components/Timeline/TimelineCards/TimelineCard.tsx:31]: **Always** add default to switch.

File: src/components/Timeline/TimelineCards/TimelineCard.tsx [hunk 6]
```diff
@@ -23,69 +23,89 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
...
+      s.cdTimelineBlock,
+      { [s.reverse]: id % 2 === 0 }
     )
 
-    const cardContainer = classNames(
-        s.cardContainer,
-        { [s.showBack]: showBack }
+    const cardContainerStyle = classNames(
+      s.cardContainer,
+      { [s.showBack]: showBack }
```
Comment (tastytea-github) [inline src/components/Timeline/TimelineCards/TimelineCard.tsx:46]: Show back what? Back button? Then pls call it "showBackButton".

File: src/components/Timeline/TimelineCards/TimelineCard.tsx [hunk 7]
```diff
@@ -23,69 +23,89 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
...
+    let cardSize;
+
+    switch (cardInfo.size) {
+        case 'xs': cardSize = s.xs
+            break
+        case 'sm': cardSize = s.sm
+            break
+        case 'md': cardSize = s.md
+            break
+        case 'lg': cardSize = s.lg
```
Comment (tastytea-github) [inline src/components/Timeline/TimelineCards/TimelineCard.tsx:58]: default

File: src/components/Timeline/TimelineCards/TimelineCard.tsx [hunk 8]
```diff
@@ -23,69 +23,89 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
...
+            break
+    }
+
+    const cdTimelineContentStyle = classNames(
+      s.cdTimelineContent,
```
Comment (tastytea-github) [inline src/components/Timeline/TimelineCards/TimelineCard.tsx:65]: cdTimelineContent - what is cd? would be nice to set a clear name.

**Review by tastytea-github — APPROVED** [review]

**Pull request merged**